.github/workflows/tox_verify.yaml

```diff
@@ -139,14 +139,16 @@ jobs:
         run: uvx --with=tox-uv tox -e pytest
 
   tests_deployed:
-    name: Fill tests, deployed, ${{ matrix.os }}, ${{ matrix.python }}
+    name: Fill tests, deployed, ${{ matrix.name }}, ${{ matrix.python }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         include:
-          - os: ubuntu-latest
+          - os: [self-hosted-ghr, size-xl-x64]
```

Comment:
With this change, matrix.os was showing up as `Array`. Happy to find a more dynamic way display a descriptive name though!

Reply:
I think it's perfectly good 👍